source/components/NPSCollect/__tests__/NPSCollect-test.js

```diff
@@ -150,6 +151,33 @@ describe('NPSCollect container component', () => {
     assert(wrapper.state('sending') === false)
     assert(wrapper.state('submittedScore'))
   })
+
+  describe('sets params', () => {
+    let handleScoreSelected
+
+    before(() => {
+      handleScoreSelected = sinon.spy()
+      sinon.stub(NPSCollect.prototype, 'handleScoreSelected', handleScoreSelected)
+
```

Comment:
This is a _massive_ deal.  Is this line break is intended? (does it not break the linting?)